examples/server/bench/script.js

```diff
@@ -0,0 +1,84 @@
+import http from 'k6/http';
+import { check, sleep } from 'k6';
+import { SharedArray } from 'k6/data';
+import { Counter, Gauge, Rate } from 'k6/metrics';
+
+const data = new SharedArray('conversations', function () {
+    return JSON.parse(open('./ShareGPT_V3_unfiltered_cleaned_split.json'))
+
+        // Filter out the conversations with less than 2 turns.
+        .filter(data => data["conversations"].length >= 2)
+        // Only keep the first two turns of each conversation.
+        .map(data => Array(data["conversations"][0]["value"], data["conversations"][1]["value"]));
```

Comment:
I think downloading a 600MB dataset and only use 2 messages in each conversation is kinda wasting. Can we maybe just take a smaller dataset? Like this one: https://github.com/nas5w/imdb-data

Another idea is simply generate random string, because we don't care about the result anyway.

Reply:
explained in the PR description: the idea is to reuse what is widely accepted in [vLLM](https://github.com/vllm-project/vllm/blob/1cb0cc2975d1c42c445c795f955b783e78919502/benchmarks/benchmark_serving.py#L63) in order to ease comparisons